libnetdata/queue/README.md

```diff
@@ -0,0 +1,15 @@
+<!--
+title: "Queue"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/libnetdata/queue/README.md
+-->
+
+# Queue
+
+Queue is a thread-safe library to handle queue processes with independent objects.
+
+Library includes functions below:
```

Comment:
@erdem2000 Could you add a description for each of these functions?

Reply:
Sure, planning to add them within today.

Reply:
@kickoke there's no value in documenting these functions because they are not meant for end-users. Intent is clear from the API and every engineer knows what a queue is.

Reply:
@vkalintiris I know that this isn't docs for end users. However, we do need a place for our internal documentation, as well, and this place seems appropriate. I don't care if we use this repository or confluence, or another tool for internal documentation. 
I do see a value in capturing this knowledge, even if it seems trivial. One of the benefits is, that we can onboard new engineers easier. Given that we expect further growth, this should be a good motivation for us to start improving our internal documentation. 

What do you think?